src/debug/ee/controller.cpp

```diff
@@ -7792,31 +7860,33 @@ bool DebuggerJMCStepper::TrapStepInHelper(
         // So in either case, we have to execute the rest of this function.
     }
 
-    MethodDesc * pDesc = pInfo->m_activeFrame.md;
-    DebuggerJitInfo *dji = NULL;
+    if (!fIsJump)
```

Comment:
Thanks for fixing this : ) 👍

Reply:
No problem! By the way, these changes are the same as #26781.